### ISSUE TITLE
Thank You Post

### ISSUE THREAD

**Issue by FeanTMX** [nus-cs2030/2122-s2#560]
Not exactly an issue but just wanted to use this place to thank the teaching team for their hard work this semester! Thank you prof (idk if you will see this) and all the TAs! CS2030 is a tough module but the teaching team made it better! THANK YOU!!!!!

**Comment by fionaxjy** [comment]
Thank you Prof and TAs! It has been a tough semester but the resources and dedication y'all put into the module made the learning experience a lot more enjoyable.

**Comment by jodytng** [comment]
thank you prof and TAs (special mention to my TA mingjiang) for helping us so much along the way!!! cs2030 was a really tough module for me but im really glad we all pulled through this together :))))

**Comment by blmj00** [comment]
thanks prof and TAs(shoutout to javier for explaining concepts in actual english) for guiding and always being there for us when we have any questions!

**Comment by dorothyyuann** [comment]
thank you prof for always being so passionate and making lectures interesting! I really appreciated the use of analogies in explaining computing concepts. And to my TA Joel Wong (subscribe him on youtube) who took the time out of labs to reply my incessant questions...even when they're all missing symbol errors and wrongly added <T> :") this mod was a steep learning curve but i took away a lot!

**Comment by AaronTay21** [comment]
Thanks prof and TAs for the help and assistance!

**Comment by jonyeokj** [comment]
Thank you profs and TAs, this mod was the most interesting and fun to take thus far! The lectures and lab were especially enjoyable and I appreciate the effort y'all put into making lessons more engaging and rewarding. Most notably, thank you Wei Yang for being such a patient and dedicated TA, helping and guiding us through the semester with our labs, PA assessments and practice!

**Comment by jwjiawei** [comment]
yup Thank you profs and TAs for all your hardwork and effort for doing your best for us :)

**Comment by Zarilex09** [comment]
Thank you Prof and TAs. sorry that i, so dumb but your lectures are really clear and engaging, can really see your passion for teaching. I just take a longer time to grasp coding concepts.

**Comment by koh-zhe-wei** [comment]
thank you profs and TAs!!

**Comment by tanweiming00** [comment]
Thanks the teaching team! Hope everyone would move on from this module.

**Comment by coffeecode2** [comment]
thank you profs and TAs for the help and guidance throughout this mod!!

**Comment by saxyrinchen** [comment]
Thank you Prof and TAs (special mention to MT for replying to my queries so quickly everytime!) for the constant support provided to us throughout the course of this module! Also thank u fellow 2030 course mates for the endless discussions (failing project test cases, debugging labs, etc) on GitHub, saving us a lot of precious time!

**Comment by jkjliang** [comment]
Thank you prof and TAs for all the help and guidance thus far!! This mod has been extremely difficult but thank you all for being so passionate and answering our endless queries so quickly, it really did help in my understanding significantly!!

**Comment by Lu-Yi-Fan** [comment]
Thank you profs and TAs!

**Comment by sshawnlim** [comment]
Thank you Prof and TAs for all the guidance along the way!! The Labs and PAs were tough but y'all were always available to help. Appreciate the hard work!!

**Comment by Zhenyubbx** [comment]
Thank you profs and TAs!

**Comment by kayyenl** [comment]
Thank you all Teaching Staffs!

**Comment by calvinseptyanto** [comment]
Big props to all the teaching staffs!

**Comment by yangyee-hub** [comment]
Thank you profs and TAs! And the kind contribution of various students in this github making the learning a fun ane enjoyable one.

**Comment by AmyXueYG** [comment]
Thank you profs and TAs for the help and guidance!

**Comment by kszechong** [comment]
Thank you prof + TAs for everything! This mod oozes passion and effort :))

**Comment by hx1228** [comment]
thank you profs and my brilliant TA for all the guidances along the way!
